src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,61 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+
+Given an ambient dimension and a subspace dimension return the ideal
+generated by the Plücker relations in a polynomial ring over the rationals. 
+
+# Examples
+```jldoctest
+julia> grassman_pluecker_ideal(2, 4)
+ideal(x[0]*x[5] - x[1]*x[4] + x[2]*x[3])
+```
+"""
+function grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+    return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+end
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(ring::MPolyRing, subspace_dimension::Int, ambient_dimension::Int)
```

Comment:
One way to merge the two docstrings would be to change this to
```suggestion
    grassman_pluecker_ideal([ring::MPolyRing, ]subspace_dimension::Int, ambient_dimension::Int)
```
or to
```suggestion
    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
    grassman_pluecker_ideal(ring::MPolyRing, subspace_dimension::Int, ambient_dimension::Int)
```
And then of course adjust the following test and the examples

Reply:
(again, you can do this now if you like, but we can also squash-merge it as is and you change it later (or leave it as it is, whatever), as you prefer